source/fundamentals/encrypt-fields.txt

```diff
@@ -0,0 +1,73 @@
+==============
+Encrypt Fields
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can encrypt fields in a document using a feature called
+**client-side encryption**.
```

Comment:
since we call it In-Use encryption in the other docs we should also use that here.  Maybe call it a set of features called In-Use Encryption so that it references more than one?